world_universities_and_domains.json

```diff
@@ -25290,13 +25290,13 @@
   },
   {
     "web_pages": [
-      "http://www.durhamc.on.ca/"
+      "https://durhamcollege.ca/"
     ],
     "name": "Durham College",
     "alpha_two_code": "CA",
     "state-province": null,
```

Comment:
It would be nice if you provide the state-province data...

Reply:
Is this a new requirement now?

Reply:
It is just nice to have the information filled in.
If you know the info about Durham college you might as well know its state-province, so what is the harm in filling that info in?